docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites.md

```diff
@@ -76,302 +82,44 @@ Installing YugabyteDB Anywhere on Airgapped hosts, without access to any Interne
 
 ### Kubernetes-based installations
 
-For a [Kubernetes-based installation](#kubernetes-based-installations), you need to ensure that the host can pull container images from the [Quay.io](https://quay.io/) container registry. For details, see [Pull and push YugabyteDB Docker images to private container registry](#pull-and-push-yugabytedb-docker-images-to-private-container-registry). 
+The YugabyteDB Anywhere Helm chart has been tested using the following software versions:
+
+- Kubernetes 1.22
+- Helm 3.10
 
-In addition, you need to ensure that core dumps are enabled and configured on the underlying Kubernetes node. For details, see [Specify ulimit and remember the location of core dumps](#specify-ulimit-and-remember-the-location-of-core-dumps). 
+
+Before installing YugabyteDB Anywhere, verify that you have the following:
+
+- A Kubernetes cluster configured with [Helm](https://helm.sh/).
+- A Kubernetes secret obtained from [Yugabyte](https://www.yugabyte.com/platform/#request-trial-form).
+
+In addition, ensure the following:
+
+- The host can pull container images from the [quay.io](https://quay.io/) container registry. If the host cannot do this, you need to prepare these images in your internal registry by following instructions provided in [Pull and push YugabyteDB Docker images to private container registry](../prepare-environment/kubernetes#pull-and-push-yugabytedb-docker-images-to-private-container-registry). 
+- Core dumps are enabled and configured on the underlying Kubernetes node. For details, see [Specify ulimit and remember the location of core dumps](#specify-ulimit-and-remember-the-location-of-core-dumps).
+- You have the kube-state-metrics add-on version 1.9 in your Kubernetes cluster. For more information, see [Install kube-state-metrics](../prepare-environment/kubernetes#install-kube-state-metrics).
+- A load balancer controller is available in your Kubernetes cluster.
+- A StorageClass is available with the SSD and `WaitForFirstConsumer` preferably set to `allowVolumeExpansion`. For more information, see [Control placement of YugabyteDB Anywhere pod](../install-software/kubernetes/#control-placement-of-yugabytedb-anywhere-pod).
 
 #### Specify ulimit and remember the location of core dumps
```

Comment:
we should move this section (specify ulimit and remember) to the "prepare" section because it is not a pre-req, it describes how to prepare the underlying environment. The statement above asking them to ensure core dumps are enabled should be sufficient on this page.

Reply:
Moved section